CaptureImage.py

```diff
@@ -0,0 +1,47 @@
+import os
+import cv2
+
+def main() :
+
+	gestureClass = input("Enter the type of gesture")
+
+	cap = cv2.VideoCapture(0)
+
+	rootName = 'HandGestures/' + gestureClass
+
+	try:
+		if(not os.path.exists(rootName)):
+			os.makedirs(rootName)
+	except OSError:
+		print('Cannot create the directory')
+
+	currentFrame = 0
+
+	while(True):
+
+		if(not cap.isOpened()):
+			cap.open()
+
+		ret, frame = cap.read()
+
+		frame = cv2.resize(frame, (128, 128)) 
+
+		cv2.imshow('frame', frame)
+
+		path = "./"+rootName
+
+		print(path)
```

Comment:
No need of printing the path. If you strictly need to log the path, use the `logging` module instead.